orbit/cmd/desktop/desktop.go

```diff
@@ -192,6 +194,46 @@ func main() {
 			migrateMDMItem.Hide()
 		}
 
+		// TODO: we can probably extract this into a function that sets up both the migrator and the
```

Comment:
Is this worth doing now? I'd say yes but not too big a deal.

Reply:
@jahzielv, do you want to give that a try as follow up in the feature branch?

Reply:
Sure!